addons/point_of_sale/tools/posbox/configuration/odoo.conf

```diff
@@ -7,3 +7,4 @@ pidfile = /var/run/odoo/odoo.pid
 limit_time_cpu = 600
 limit_time_real = 1200
 max_cron_threads = 0
+server_wide_modules=hw_drivers,hw_escpos,hw_posbox_homepage,point_of_sale,web
```

Comment:
is there really a point to have `point_of_sale` as it just contains shell scripts and config file ? I do guess it is needed in the code as if it is not in the loaded modules the code won't load the file in it ?

Reply:
yes you might be right, loading `point_of_sale` looks like an old convention at this point which has no reason to be, but I won't change this now as the new image release is already complex enough